integrationtest/mapper/pojo-standalone-realbackend/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/standalone/realbackend/mapping/VectorFieldIT.java

```diff
@@ -57,6 +59,14 @@ class VectorFieldIT {
 	public StandalonePojoMappingSetupHelper setupHelper = StandalonePojoMappingSetupHelper.withSingleBackend(
 			MethodHandles.lookup(), BackendConfigurations.simple() );
 
+	@BeforeAll
+	static void beforeAll() {
+		assumeTrue(
```

Comment:
I've added this assumption to address the main build failure  🙈

Reply:
Maybe use a related Jira issue, though?
Backporting becomes quite hard once you start attaching unrelated stuff to Jira issues :)